test/index.js

```diff
@@ -160,4 +162,19 @@ HmJx6deP0rullujAEXAf0QRFq39TiLld//8aIx/ZSg==
       done()
     })
   })
+  
+
+})
+
+describe('getSSLCertificate.get() live tests', function() { 
+  var timeoutUrl = 'singnet.com.sg'
+  
+  // Timeout test
+  it('should reject with an error if the timeout is exceeded', function(done) {
+    getSSLCertificate.get(timeoutUrl, {timeout: 500}).catch(function(err) {
```

Comment:
This test case if failing on Node 9. This is a live test so our options here is to use something like `nock` to mock this request to make it more deterministic, or set it to a really ridiculous amount like `1ms`.